accesshandler/pkg/providers/aws/flask/testdata/argschema.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This argschema looks like it needs to be updated - it should be used in tests to check that the schema of the provider matches what we expect.